storage/postgres/visibility.go

```diff
@@ -29,7 +35,30 @@ type visibilityStorage struct {
 func (vs *visibilityStorage) Create(ctx context.Context, visibility *types.Visibility) (string, error) {
 	v := &Visibility{}
 	v.FromDTO(visibility)
-	return create(ctx, vs.db, visibilityTable, v)
+	id, err := create(ctx, vs.db, visibilityTable, v)
+	if err != nil {
+		return "", err
+	}
+	return id, vs.createLabels(ctx, id, visibility.Labels)
```

Comment:
are the 2 queries executed in transaction?

Reply:
It might be somewhere else, I just cannot find it

Reply:
Yes, they are - https://github.com/Peripli/service-manager/blob/query-interpreter/api/visibility/visibility_controller.go#L58